valhalla/baldr/graphtile.h

```diff
@@ -326,6 +326,12 @@ class GraphTile : public boost::intrusive_ref_counter<GraphTile, GraphTileRefCou
    */
   EdgeInfo edgeinfo(const size_t offset) const;
```

Comment:
are there any places in the code left that use this? i was trying to think of some in my head and my thought was that it happens in mjolnir somewhere but i couldnt come up with the exact location.

Reply:
I replaced all places where the old one overload is used(see new commit). Only a few `GraphTile` methods are left that uses this overload: `GraphTile::GetNames` and `GraphTile::GetTypes`
So if it's okay to "break" `GraphTile` api i would do one of the two things
1. Remove this overload and fix ^ 2 methods.
2. Move this overload to the protected/private section